xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -0,0 +1,1834 @@
+       <para>
+         Reboot the compute nodes, individually or in groups, using the
+         <filename>ardana-reboot.yml</filename> playbook as follows:
+       </para>
+       <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+    &prompt.ardana;ansible-playbook -i hosts/verb_hosts ardana-reboot.yml --limit ardana-cp-slcomp0001,ardana-cp-slcomp0002
+
+PLAY [all] ********************************************************************
+
+TASK: [setup] *****************************************************************
+ok: [ardana-cp-slcomp0001]
+ok: [ardana-cp-slcomp0002]
+
+PLAY [localhost] **************************************************************
+
+TASK: [pbstart.yml pb_start_playbook] *****************************************
+ok: [localhost] => {
+    "msg": "Playbook started - ardana-reboot.yml"
+}
+
+msg: Playbook started - ardana-reboot.yml
+
+...
+
+PLAY [localhost] **************************************************************
+
+TASK: [pbfinish.yml pb_finish_playbook] ***************************************
+ok: [localhost] => {
+    "msg": "Playbook finished - ardana-status.yml"
+}
+
+msg: Playbook finished - ardana-status.yml
+
+PLAY [localhost] **************************************************************
+
+TASK: [pbfinish.yml pb_finish_playbook] ***************************************
+ok: [localhost] => {
+    "msg": "Playbook finished - ardana-reboot.yml"
+}
+
+msg: Playbook finished - ardana-reboot.yml
+
+PLAY RECAP ********************************************************************
+ardana-cp-slcomp0001       : ok=120  changed=11   unreachable=0    failed=0
+ardana-cp-slcomp0002       : ok=120  changed=11   unreachable=0    failed=0
+localhost                  : ok=27   changed=1    unreachable=0    failed=0
+  </screen>
+     </step>
+   </procedure>
+   <important>
+     <para>
+       You must ensure that there is sufficient unused workload capacity to
+       host any migrated workload or Amphora instances that may be running on
+       the targeted compute nodes.
+     </para>
+     <para>
+       When rebooting multiple compute nodes at the same time, consider
+       manually migrating any running workloads and Amphora instances off the
+       target nodes in advance, to avoid any potential risk of workload or
+       service interruption.
+     </para>
+   </important>
+ </section>
+
+ <section xml:id="post-upgrade-tasks">
+   <title>Post-Upgrade Tasks</title>
+   <para>
+     After the cloud has been upgraded to &productname; &productnumber; &clm;,
+     if <systemitem class="service">designate</systemitem> was previously configured, then the deprecated service
+     components, <literal>designate-zone-manager</literal> and
+     <literal>designate-pool-manager</literal>, were being used.
+   </para>
+   <para>
+     They will continue to operate correctly under &productname; &productnumber; &clm;,
+     but we recommend that you migrate to using the newer <literal>designate-worker</literal>
```

Comment:
Here we wrapped "designate-worker" and "designate-producer" as literals, but earlier they were wrapped as systemitem class=service... Not sure which is correct...

Reply:
I favour the [systemitem] form -- it is more specific, less general.